web/tests/functional/authentication/test_authentication.py

```diff
@@ -195,6 +195,8 @@ def test_nonauth_storage(self):
             subprocess.check_output(
                 store_cmd, encoding="utf-8", errors="ignore")
 
+    # TODO: This test should not test case-insensivity, only the successful
+    # group authorization. The permission tests should do that.
```

Comment:
Do we still need this test case? If your new test cases test this, then we can remove it.

Reply:
Two things are tested in one testcase. But it does not cause problem. Comment removed.